src/testUtils/java/org/broadinstitute/hellbender/testutils/VariantContextTestUtils.java

```diff
@@ -26,20 +24,17 @@
 import org.broadinstitute.hellbender.utils.variant.GATKVariantContextUtils;
 import org.testng.Assert;
 
-// This should be:
-//import org.apache.logging.log4j.LogManager;
-//import org.apache.logging.log4j.Logger;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
-
 import java.io.File;
 import java.io.PrintStream;
 import java.util.*;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
+// This should be:
+//import org.apache.logging.log4j.LogManager;
```

Comment:
What's the deal with this change?  can we remove these comments?

Reply:
Because Intellij likes to auto resolve / update the import statements, I've found that the loggers occasionally get messed up and are switched to be the other includes which don't work.

I added this comment here so that I could preserve which loggers are correct.  I'll reword the comment and make it clear.  Initially this was above the logger lines, but since Intellij auto-organizes the imports they're no longer above the `logging` imports.